settings.gradle

```diff
@@ -1,3 +1,23 @@
 rootProject.name = 'apoc'
 
-include("extended")
\ No newline at end of file
+include("extended")
+
+include ':core', ':common', ':processor', ':test-utils'
+project(':core').projectDir = new File(rootDir, 'apoc-core/core')
+project(':common').projectDir = new File(rootDir, 'apoc-core/common')
+project(':processor').projectDir = new File(rootDir, 'apoc-core/processor')
+project(':test-utils').projectDir = new File(rootDir, 'apoc-core/test-utils')
+
+startParameter.excludedTaskNames = [
+        ':core:build',
+        ':common:build',
+        ':processor:build',
+        ':test-utils:build',
+        ':core:test',
+        ':common:test',
+        ':processor:test',
+        ':test-utils:test',
+        ':core:publish',
+        ':common:publish',
+        ':test-utils:publish'
```

Comment:
There is no 'processor:test' and also no mention of 'test-startup' which was included in the CI.yaml. Is that intentional?